<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AES Encryption/Decryption Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   
   <!-- Encryption Section -->
   <widget class="QLabel" name="label_input_file">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Input File:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_input_file">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_browse_input">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>

   <widget class="QLabel" name="label_output_file">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Output File:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_output_file">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_browse_output">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>

   <widget class="QLabel" name="label_aes_key">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Save AES Key As:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_aes_key">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_browse_aes_key">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButton_encrypt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>

   <!-- Decryption Section -->
   <widget class="QLabel" name="label_decrypt_input_file">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Encrypted File:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_decrypt_input_file">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_browse_decrypt_input">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>230</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>

   <widget class="QLabel" name="label_decrypt_output_file">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Output File:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_decrypt_output_file">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_browse_decrypt_output">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>280</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>

   <widget class="QLabel" name="label_load_aes_key">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Load AES Key:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_load_aes_key">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_browse_load_aes_key">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButton_decrypt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>380</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>

  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
